games-board/cockatrice/cockatrice-20200823.ebuild

```diff
@@ -38,32 +39,34 @@ BDEPEND="
 	dev-qt/linguist-tools:5
 	client? ( dev-libs/protobuf )
 	server? ( dev-libs/protobuf )
+	test? ( dev-cpp/gtest )
 "
 DEPEND="${RDEPEND}"
 
-# As the default help/about display the sha1 we need it
-SHA1='294b433'
-
 S="${WORKDIR}/Cockatrice-${MY_PV}"
 
-PATCHES=(
-	"${FILESDIR}"/use-ccache.patch
-	"${FILESDIR}"/${P}-qt-5.15.patch
-)
-
 src_configure() {
 	local mycmakeargs=(
 		-DBUILD_SHARED_LIBS=OFF
 		-DUSE_CCACHE=OFF
 		-DWITH_CLIENT=$(usex client)
 		-DWITH_ORACLE=$(usex oracle)
 		-DWITH_SERVER=$(usex server)
+		-DTEST=$(usex test)
 		-DICONDIR="${EPREFIX}/usr/share/icons"
 		-DDESKTOPDIR="${EPREFIX}/usr/share/applications" )
 
 	# Add date in the help about, come from git originally
-	sed -e 's/^set(PROJECT_VERSION_FRIENDLY.*/set(PROJECT_VERSION_FRIENDLY \"'${SHA1}'\")/' \
+	sed -e 's/^set(PROJECT_VERSION_FRIENDLY.*/set(PROJECT_VERSION_FRIENDLY \"'${MY_PV}'\")/' \
 		-i cmake/getversion.cmake || die "sed failed!"
 
 	cmake_src_configure
 }
+
+pkg_postinst() {
+	xdg_icon_cache_update
+}
+
+pkg_postrm() {
+	xdg_icon_cache_update
+}
```

Comment:
You're using these xdg functions without inheriting the `xdg-utils` eclass. You should just inherit the `xdg` eclass though so you don't have to define these functions at all. If you order it `xdg cmake` rather than `cmake xdg` then you can avoid a bad interaction between the two.

Reply:
Hey @chewi thank you for the feedback, I have upgraded the branch.